-From: Harry Gibson <harry.gibson@smoothwall.net>
Date: Wed, 26 Jun 2017 11:09:05 +0100
Description: Disable autoupdate as it would blow over
 our changes to the agent

Patch-Name: disableautoupdate.patch
---

--- a/config/waagent.conf
+++ b/config/waagent.conf
@@ -86,7 +86,7 @@
 # OS.EnableRDMA=y
 
 # Enable or disable goal state processing auto-update, default is enabled
-# AutoUpdate.Enabled=y
+AutoUpdate.Enabled=n
 
 # Determine the update family, this should not be changed
 # AutoUpdate.GAFamily=Prod
